CONTRIBUTING.md

```diff
@@ -29,6 +29,48 @@ To contribute a change, please:
 6. If changes are requested, the PR author should address the comments, and then
    ask for review again.
 
+### Developing the cookiecutter template
+
+We quite like installing cookiecutter as a [uv tool](https://docs.astral.sh/uv/guides/tools/), so it's available globally:
```

Comment:
It's nice you're using my preferred syntax for links, but now this file isn't self-consistent

```suggestion
We quite like installing cookiecutter as a [uv tool](https://docs.astral.sh/uv/guides/tools), so it's available globally:
```